roles/installer/templates/tower_secret_key.yaml.j2

```diff
@@ -10,4 +10,4 @@ metadata:
     app.kubernetes.io/managed-by: awx-operator
     app.kubernetes.io/component: awx
 stringData:
-  secret_key: '{{ lookup('password', '/dev/null length=32 chars=ascii_letters,digits') }}'
+  secret_key: '{{ tower_custom_secret_key | lookup('password', '/dev/null length=32 chars=ascii_letters,digits') }}'
```

Comment:
This seems like it would cause an error if `tower_custom_secret_key` is not set?